sass/true/_messages.scss

```diff
@@ -46,25 +16,20 @@ $-tnl: '\a ';
 /// @require {function} _true-str-split
 @mixin _true-message(
   $message,
-  $output: comments
+  $output: 'comments'
```

Comment:
I disagree. One mixin with a clear argument is more direct than multiple mixins named after possible arguments.